application-interactive-maps-ui/src/main/resources/Maps/Code/MapSheet.xml

```diff
@@ -459,13 +459,9 @@ require(['jquery', 'leaflet', 'leaflet-commons', 'leaflet-indoor', 'bootstrap'],
       $('.map-search-filter').removeClass('open');
       leafletCommons.updateURLWithMapState();
     });
-
+    $(init).on('xwiki:dom:updated', init);
```

Comment:
This statement is inside the `init` function. It won't work because the `init` function will never be called and even if it's called like this, it will result in an infinite recursion.

You can move this statement outside the scope of the `init` function and it should work just fine.

Reply:
@9inpachi is right. This line needs to be moved below, outside the ``init`` function.